DaleCS/TheITCrowd-Team8-172
# 001: User Registration

## User Story Description
As a user, I need to be able to register on the app so that I can make an account that only I can access.

## Additional Notes
This covers both frontend and backend. We need backend API for accepting email, password, and username to create a user in our database. Password needs to be hashed w/ SHA256 first before storing in the database. Frontend aspect also needs to be intuitive.

## How to Test
Run both the frontend and backend then open up a browser and go to localhost:3000/register and test user registration UI. Then check your MySQL database 'users' table to see if the new user has been created.
